src/graphql/helpers.ts

```diff
@@ -1,8 +1,9 @@
 import { jsonParse } from '../helpers/utils';
+import space from './operations/space';
```

Comment:
This import is not used. Added automatically by some auto-completion feature in your IDE or so maybe?

```suggestion
```

Reply:
Rest looks good btw.

Reply:
@mktcode I believe that your hypothesis is correct and I didn't notice...